### DISCUSSION TITLE
Lastfm scrobbles in Navidrome

### DISCUSSION THREAD

**Discussion by isle9** [navidrome/navidrome#2339] [Ideas]
Would it be possible to show lastfm scrobbles for songs inside Navidrome?
For instance, in a column?
It would be great if we were also able to use them inside smart playlists.

**Comment by deluan** [comment]
Not sure if this is a good idea. Last.fm is very unstable, and it is degrading year after year, unfortunately.

**Reply by isle9** [reply]
> Not sure if this is a good idea. Last.fm is very unstable, and it is degrading year after year, unfortunately.

Oh really? I wasn't aware of that.
You can supplement Last.fm for any other service like ListenBrainz etc.
